cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -652,7 +656,8 @@
 
     public static final int DEFAULT_TIME_SERIES_TABLE_ROLL_TIME = 3600;
 
-    public static final long DEFAULT_RETENTION_HOURS = 2;
+    public static final long DEFAULT_SECOND_RETENTION_HOURS = 2;
```

Comment:
Don't define defaults in `Constants`. They should be defined in `cdap-default.xml`

Reply:
Noticed they are already defined in `cdap-default.xml`, removing them